cyberark/conjur
# updated PDF of Contributor Agreement

Adds updated PDF of Contributor Agreement
Closes [404](https://github.com/cyberark/conjur/issues/404)

#### What does this pull request do?
Updates PDF of Contributor Agreement
#### What background context can you provide?
Lawyers updated doc.
#### Where should the reviewer start?
homepage
#### How should this be manually tested?
Clicl on "Contributing" in the left sidebar (will bring you to GH) then click on link to PDF and make sure it's the newest version. 
Note: 2. should read "2. Copyright Assignment, License and Waiver."  (used to read: **lic** and waiver)
#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/updated-contributor-agreement/
#### Questions:
> Does this have automated Cucumber tests?
no

> Can we make a blog post, video, or animated GIF out of this?

no
> Is this explained in documentation?
no

> Does the knowledge base need an update?
no
